doc/usage/configuration.rst

```diff
@@ -1644,7 +1645,8 @@ the `Dublin Core metadata <http://dublincore.org/>`_.
 Options for LaTeX output
 ------------------------
 
-These options influence LaTeX output. See further :doc:`latex`.
+These options influence LaTeX output. Refer to :doc:`/latex` for more
```

Comment:
why not move `latex.rst` also to `usage`?

Reply:
Yup, I plan to move this but this is only part 4. I'm moving stuff one by one so it's possible to review everything. I will move this in part 5, which moves everything related to builders (I have the changes sitting in my local repo)